agent-control/agent-type-registry/newrelic/com.newrelic.opentelemetry.collector-0.1.0.yaml

```diff
@@ -0,0 +1,177 @@
+      # See https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckv2extension/README.md#configuration
+      # For more details
+      path:
+        description: "path to health check endpoint"
+        type: string
+        required: false
+        default: "/health/status" # Taken from example at: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/extension/healthcheckv2extension/README.md#configuration
+      port:
+        description: "port to health check endpoint"
+        type: number
+        required: false
+        default: 13133
+  k8s:
+    chart_values:
+      nr-k8s-otel-collector:
+        description: "Newrelic otel collector chart values"
+        type: yaml
+        required: false
+        default: {}
+      global:
+        description: "Global values for the chart"
+        type: yaml
+        required: false
+        default: {}
+    chart_version:
+      description: "Newrelic otel collector chart version"
+      type: string
+      required: true
+    chart_name:
+      description: "nr-k8s-otel-collector chart name"
+      type: string
+      required: false
+      default: "nr-k8s-otel-collector"
+    chart_repository:
+      url:
+        description: "chart repository url"
+        type: string
+        required: false
+        default: "https://helm-charts.newrelic.com"
+        variants:
+          ac_config_field: "chart_repository_urls"
+          values: ["https://helm-charts.newrelic.com"]
+      secret_reference:
+        description: "HelmRepository secret reference (secretRef)"
+        type: yaml
+        required: false
+        default: null
+      certificate_secret_reference:
+        description: "HelmRepository certificate secret reference (certSecretRef)"
+        type: yaml
+        required: false
+        default: null
+deployment:
+  on_host:
+    health:
+      interval: 5s
+      initial_delay: 5s
+      timeout: 5s
+      http:
+        path: "${nr-var:health_check.path}"
+        port: ${nr-var:health_check.port}
+    version:
+      path: /usr/bin/nrdot-collector-host
+      args: -v
+      regex: \d+\.\d+\.\d+
+    filesystem:
+      otel-config:
+        config.yaml: |
+          ${nr-var:config}
+    executables:
+      - # Important to note the binary name is nrdot-collector-host matching the new nrdot binary
+        id: nrdot-collector-host
+        path: /usr/bin/nrdot-collector-host
+        args: >-
+          --config=${nr-sub:autogenerated_agent_dir}/otel-config/config.yaml
+          --feature-gates=-pkg.translator.prometheus.NormalizeName
+        env:
+          # sets the otel-collector "env" source resource detector
+          OTEL_RESOURCE_ATTRIBUTES: "host.id=${nr-ac:host_id}"
+        restart_policy:
+          backoff_strategy:
+            type: fixed
+            backoff_delay: ${nr-var:backoff_delay}
+  # See com.newrelic.infrastructure Agent type for description of fields.
```

Comment:
I've just noticed this comment, copy&pasted for sure but it is a bit misleading, isn't it?